designer/server/src/test/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceSpec.scala

```diff
@@ -308,7 +307,7 @@ class DeploymentServiceSpec
 
     val activities = dbioRunner.run(activityRepository.findActivities(processIdWithName.id)).futureValue
 
-    activities.size shouldBe 3
+    activities.size shouldBe 2
```

Comment:
please confirm this is ok. TBH I don't understand why the "activities" part was touched in this PR

Reply:
Ahh, I forgot to mention about that because I thought that Michał would review this part of changes and he will now what's going on. Test was checking if deployment manager can add activity. This checking was incorporated into test case that was checking something else. Feature allowing adding activities by deployment managers wasn't used anywhere so I removed this feature and related noice